modules/effects/src/effect_notification.ts

```diff
@@ -46,7 +46,7 @@ function getEffectName({
 }: EffectNotification) {
   const isMethod = typeof sourceInstance[propertyName] === 'function';
 
-  return `"${sourceName}.${propertyName}${isMethod ? '()' : ''}"`;
+  return `"${sourceName}.${String(propertyName)}${isMethod ? '()' : ''}"`;
```

Comment:
Why this change required? I thought Typescript knew that `propertyName` will be transformer to string under the hood

Reply:
Good question @L2jLiga ! I'm bringing the `propertyName` back to the actual allowed property types, which are `string | number | Symbol`. 
`Symbol` cannot be implicitly converted to string, and needs some help.

I probably should rename `propertyName` to the `propertyKey` to be more descriptive. I'll leave it out for later. Thanks for asking!